src/app/footer/footer.component.html

```diff
@@ -149,28 +149,29 @@
           <ul class="footer-list m-0 affiliation-list">
             <li class="m-0 title">Find us on</li>
             <li class="m-0">
-              <a [href]="Dockstore.DOCKSTORE_REPO" target="_blank" rel="noopener noreferrer" class="footer-link">
-                <img
-                  class="site-icons-tab hidden-sm"
-                  src="../assets/images/dockstore/github-mark-light.png"
```

Comment:
This appears to be the only place that uses the png and now it's removed

Reply:
@garyluu Yeah so I deleted the png